python/simulation/fake_raft.py

```diff
@@ -0,0 +1,509 @@
+"""
+Makes a raft's worth of image files based on a user-specified single-sensor
+image file, the raft designator, and the sensor type (ITL or E2V, which 
+probably could be guessable from the single-sensor image).  The test type
+probably also needs to be specified (for the file naming)
+The data in the nine output images are the same as in the input image.  What
+is new is the raft coordinate-related header keywords.  These are derived
+from the specification in LCA-13501.
+"""
+
+from __future__ import print_function, absolute_import, division
+#import os
+#import copy
+import numpy as np
+import astropy.io.fits as fits
+import os
+import yaml
+
+import fnmatch
+import lcatr.schema
+from DataCatalog import DataCatalog
+
+import siteUtils
+
+
+def sensor_type_from_id(sensor_id):
+    """  Get the sensor type ( e2v-CCD or ITL-CCD ) from the sensor_id string
+    """
+    stype =  sensor_id[0:7]
+    if stype == 'E2V-CCD':
+        stype = 'e2v-CCD'
+    return stype
+
+
+def make_datacat_path(root_folder,sensor_id,process_name,test_version="v0"):
+    """ Build the data catalog path for a particular test on a particular sensor
+
+    This appear to be <root_folder>/<sensor_type>/<sensor_id>/<process_name>/<test_version>
+    where <sensor_type> is 'e2v-CCD' or 'ITL-CCD' and is derived from <sensor_id>
+    """
+    sensor_type = sensor_type_from_id(sensor_id)
+    return os.path.join(root_folder,sensor_type,sensor_id,process_name,test_version)
+
+
+def make_outfile_path(root_folder,raft_id,sensor_id,process_name,file_string,test_version="v0"):
+    """ Build the path for an output file for a particular test on a particular sensor
+
+    For now this is <root_folder>/<raft_id>/<sensor_type>/<sensor_id>/<process_name>/<test_version>/<file_string>
+    where <sensor_type> is 'e2v-CCD' or 'ITL-CCD' and is derived from <sensor_id>    
+    """
+    sensor_type = sensor_type_from_id(sensor_id)
+    return os.path.join(root_folder,raft_id,sensor_id,process_name,test_version,file_string)
+
+
+def get_file_suffix(filepath):
+    """ Get the last part of the file path, basically everything after the sensor_id.
+        e.g., '_mean_bias_25.fits' or '_eotest_results.fits'
+    """
+    basename = os.path.basename(filepath)
+    tokens = basename.split('_')
+    s = ""
+    for t in tokens[1:]:
+        s += '_'
+        s += t
+    return s
+
+
+def get_template_files(sensor_id, process_name, pattern, 
+                       root_folder='LSST/mirror/SLAC-prod/prod',
```

Comment:
Given my comments below on the usage example in the `if __name__ == '__main__'` part, I think this would be better as `root_folder='LSST/mirror/BNL-prod/prod'`.  It would be good to give access to this through the `Raft` constructor, otherwise this is effectively hard-wired.